.circleci/config.yml

```diff
@@ -633,11 +710,536 @@ jobs:
+              then
+                python3 ./Tests/scripts/verify_base_branch_for_contribution.py $CIRCLE_BRANCH
+            fi
+  Create Instances:
+    <<: *container_config
+    resource_class: medium+
+    <<: *environment
+    steps:
+      - checkout
+      - setup_remote_docker
+      - *restore_cache
+      - *prepare_environment
+      - *restore_cache
+      - *add_ssh_keys
+      - *prepare_environment
+      - run:
+          name: Update Tests step
+          when: always
+          command: |
+            python3 ./Tests/scripts/update_conf_json.py
+            cp "./Tests/conf.json" "$CIRCLE_ARTIFACTS/conf.json"
+      - run:
+          name: Create ID Set
+          when: always
+          command: |
+            demisto-sdk create-id-set -o ./Tests/id_set.json
+      - run:
+          name: Build Content Descriptor
+          when: always
+          command: |
+            if [ -n "${GITHUB_TOKEN}" ] ;
+              then
+                # python3 release_notes.py $CONTENT_VERSION $GIT_SHA1 $CIRCLE_BUILD_NUM $SERVER_VERSION --github-token $GITHUB_TOKEN || echo "ignore errors"
+                # new release notes summary generator in packs format
+                python3 Utils/release_notes_generator.py $CONTENT_VERSION $GIT_SHA1 $CIRCLE_BUILD_NUM --output $CIRCLE_ARTIFACTS/packs-release-notes.md --github-token $GITHUB_TOKEN
+              else
+                # python3 release_notes.py $CONTENT_VERSION $GIT_SHA1 $CIRCLE_BUILD_NUM $SERVER_VERSION || echo "ignore errors"
+                # new release notes summary generator in packs format
+                python3 Utils/release_notes_generator.py $CONTENT_VERSION $GIT_SHA1 $CIRCLE_BUILD_NUM --output $CIRCLE_ARTIFACTS/packs-release-notes.md
+            fi
+      - run:
+          name: Common Server Documentation
+          when: always
+          command: |
+            ./Documentation/commonServerDocs.sh
+      - *create_artifact
+      - run:
+          name: Download Configuration
+          when: always
+          command: |
+            if ./Tests/scripts/is_ami.sh ;
+              then
+                ./Tests/scripts/download_demisto_conf.sh
+
+              else
+                ./Tests/lastest_server_build_scripts/download_demisto_conf.sh
+            fi
+      - run:
+          name: Download Artifacts
+          when: always
+          command: |
+            if ./Tests/scripts/is_ami.sh ;
+              then
+                echo "Using AMI - Not downloading artifacts"
+
+              else
+                ./Tests/scripts/server_get_artifact.sh $SERVER_CI_TOKEN
+                cp demistoserver.sh ./Tests/scripts/awsinstancetool/ansibleinstall/demistoserver.sh
+            fi
+      - run:
+          name: Collect Test List And Content Packs
+          when: always
+          command: |
+            if [ -n "${INSTANCE_TESTS}" ];
+              then
+                echo "Skipping - not running in INSTANCE_TESTS build"
+                exit 0
+            fi
```

Comment:
Yes.
Since `Create Instances` job runs in both regular commit, nightly and instance testing workflows- it needs to be there.

I removed it from all jobs that are not run in these workflows